integration_test/uninstall_test.go

```diff
@@ -75,3 +76,30 @@ func TestKrewRemove_ManifestRemovedFromIndex(t *testing.T) {
 	}
 	test.Krew("remove", validPlugin).RunOrFail()
 }
+
+func TestKrewRemove_Unsafe(t *testing.T) {
+	skipShort(t)
+	test, cleanup := NewTest(t)
+	defer cleanup()
+	test.WithEnv(constants.EnableMultiIndexSwitch, 1).WithIndex()
+	test.Krew("install", validPlugin).RunOrFailOutput()
+
+	cases := []string{
+		`../index/` + validPlugin,
+		`..\index\` + validPlugin,
+		`../default/` + validPlugin,
+		`..\default\` + validPlugin,
+		`../receipts/` + validPlugin,
+		`..\receipts\` + validPlugin,
```

Comment:
How is `/receipt/` different from `/default/`? Why is this needed?
Otherwise, let's remove it.

Reply:
receipt/ is needed because we need to point this to a receipt file. and at the moment all these paths parseable as a Receipt